packages/driver/src/cypress.ts

```diff
@@ -397,6 +411,8 @@ class $Cypress {
           return
         }
 
+        this.maybeEmitCypressInCypress('mocha', 'start', args[0])
+
         if (this.config('isTextTerminal')) {
           return this.emit('mocha', 'start', args[0])
```

Comment:
I don't know why we do `if (this.config('isTextTerminal')) {` before each mocha event is emitted. This indicates we should only emit these events when running in headless run mode, eg on CI, but I don't know why that would be the case.

Reply:
I'm finding out if we even need this at all. https://github.com/cypress-io/cypress/compare/lmiller1990/try-remove-text-terminal?expand=1